plugins/python-build/bin/python-build

```diff
@@ -1645,7 +1645,7 @@ use_homebrew_tcltk() {
     if [[ -z "$PYTHON_BUILD_TCLTK_USE_PKGCONFIG" ]]; then
       local tcltk_version="$(sh -c '. '"$tcltk_libdir"'/lib/tclConfig.sh; echo $TCL_VERSION')"
       package_option python configure --with-tcltk-libs="-L$tcltk_libdir/lib -ltcl$tcltk_version -ltk$tcltk_version"
-      package_option python configure --with-tcltk-includes="-I$tcltk_libdir/include"
+      package_option python configure --with-tcltk-includes="-I$tcltk_libdir/include/tcl-tk"
```

Comment:
This seems to only be true for the latest Homebrew package. So we can't apply this universally.
```
$ apt-file search /tcl.h
<...>
tcl8.6-dev: /usr/include/tcl8.6/tcl.h
```
```
% find / -name tcl.h 2>/dev/null
/usr/local/Cellar/tcl-tk/8.6.13_5/include/tcl-tk/tcl.h
<...>
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/usr/include/tcl.h
/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/System/Library/Frameworks/Tcl.framework/Versions/8.5/Headers/tcl.h
/Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/usr/include/tcl.h
/Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/System/Library/Frameworks/Tcl.framework/Versions/8.5/Headers/tcl.h
```